input/tar2/tar_input.go

```diff
@@ -75,6 +75,9 @@ func (i Input) Apply(destinationRoot string) <-chan error {
 			bucket := &fshash.MemoryBucket{}
 			Extract(tarReader, destinationRoot, bucket, i.hasherFactory)
 
+			// bucket processing may have created a root node if missing.  if so, we need to apply its props.
+			fs.PlaceFile(destinationRoot, bucket.Root().Metadata, reader)
```

Comment:
A few inquires:

1. Does this comment mean that bucket processing is incorrect / incomplete?
1. Reading this, I initially thought this call would place the entire tar file at the root, clobbering the fs that was just extracted. Does this call only apply metadata because `reader` is at EOF? 
1. If so, could we have a different function that explicitly only applies metadata?